CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## Unreleased
 
 - Minor: Allow configuration of account types that should not trigger notifications. (#701)
+- Bugfix: Report correct rarity and luck for pet kraken. (#703)
```

Comment:
i was gonna suggest it in the version release PR

Reply:
yeah we can do it here or in the version bump pr; don't mind either way